app/services/migration/migrators/application.rb

```diff
@@ -96,6 +96,10 @@ def call
         application.version_note = "Changes migrated from ECF to NPQ"
         application.update!(ecf_npq_application.attributes.slice(*ATTRIBUTES))
       end
+
+      run_once { backfill_ecf_ids }
```

Comment:
not sure how this will work with the run once logic, we'll find out on migration, and might need to have all in one `run_once` block, worth giving it a whirl and we'll discuss after :)

Reply:
migration run completed successfully.  👍